content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx

```diff
@@ -90,6 +104,15 @@ const PrevNext = styled(Space).attrs({
   background: ${props => props.theme.color('neutral.700')};
 `;
 
+const AlignIconFirstLineCenter = styled.div`
+  display: flex;
+  align-items: start;
+
+  .icon {
+    height: 1lh;
```

Comment:
`lh` unit is now in use everywhere we care about which is useful: https://caniuse.com/mdn-css_types_length_lh